profile/report.rkt

```diff
@@ -143,11 +145,13 @@
    "clos" clos:run))
 
 (program (main)
-  (let* ([forms (hash-keys env)]
-         [fs (~>> (forms)
-                  (sort <))])
-    (write-json (for/list ([f fs])
-                  (match-let ([(list name ms) ((hash-ref env f))])
-                    (hash 'name name 'unit "ms" 'value ms))))))
+  (define fs (~>> (env) hash-keys (sort <)))
+  (define forms-data (for/list ([f fs])
```

Comment:
Nit: perhaps `hash-keys` with `try-order?` as `#t`? And `(in-list fs)` unless you feel strongly about being sequence-generic there.

I was trying to find a way to make `in-hash` work, but it doesn't have a `try-order?` argument :(

Reply:
Looks like `try-order?` was added after 8.3 (the docs say `8.3.0.11` which I think means that it only ends up in 8.4? In the past I've seen things that say 8.2.x.y but would fail on CI for Racket 8.2. As another data point, I'm running 8.3 locally and don't have `try-order?`), so I've left a note to add it in the future if we dropped support for 8.3 at some point. Changed to `in-list` though.

Reply:
Ah, I didn't catch that. Thanks for pointing that out.